spec/models/searchworks_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe SearchworksItem do
+  let(:item_id) { SearchworksItem.new }
```

Comment:
I should also mention that you don't HAVE to use subject, it's totally optional (but sometimes useful IMO).